beta/project-griffon/using-project-griffon/versions-view.md

```diff
@@ -0,0 +1,27 @@
+# Versions Assurance View
+
+## Overview
+
+You may use the Extensions Version view in order to quickly and easily view which Platform extensions you have installed and if they are up to date in a client connected to an Assurance (also known as Project Griffon) session. 
+
+## Getting Started
+
+To use this view, complete the following steps:
+
+1. [Set up Assurance](../set-up-project-griffon.md)
+2. [Create](./#creating-sessions) and [connect](./#connecting-to-a-session) to a Assurance session
```

Comment:
Just this comment here still, since this won't work in a post-Gitbook state, which is happening.

Reply:
Good to know! :)